packages/admin/src/Pages/Actions/Concerns/CanBeDisable.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Filament\Pages\Actions\Concerns;
+
+use Closure;
+
+trait CanBeDisable
+{
+    protected bool | Closure $isDisabled = false;
+
+    public function disable(bool | Closure $condition = true): static
```

Comment:
```suggestion
    public function disabled(bool | Closure $condition = true): static
```

`disabled` here instead. This is the same name we use elsewhere in Filament.